src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -80,6 +79,14 @@
         " "
     $.equal-to ""
 
+[] > text-trimmed-empty-3
+  nop > @
+    assert-that
+      trimmed.
+        text
+          "  "
```

Comment:
@EugeneDar Do you mean here you got Stack Overflow exception?

Reply:
@Graur as I pointed out in the puzzle, crashes in all tests where you need to remove more than one space.